packages/next/build/swc/index.js

```diff
@@ -179,6 +182,10 @@ function loadNative() {
       bundle(options) {
         return bindings.bundle(toBuffer(options))
       },
+
+      parse(src, options) {
+        return bindings.parse(toBuffer(src), toBuffer(options ?? {}))
```

Comment:
As `src` is already `string`, I think conversion to buffer is not required.

Reply:
should also I remove `toBuffer` on `src` arg for minify and minifySync?